Branch-SDK/Branch-SDK/BranchActivityItemProvider.m

```diff
@@ -73,52 +63,40 @@ - (id)item {
     }
     
     // Because Facebook et al immediately scrape URLs, we add an additional parameter to the existing list, telling the backend to ignore the first click
-    NSArray *scrapers = @[@"facebook", @"twitter", @"slack", @"com.apple.mobilenotes.SharingExtension"];
+    NSArray *scrapers = @[@"Facebook", @"Twitter", @"Slack", @"com.apple.mobilenotes.SharingExtension"];
```

Comment:
What is com.apple.mobile.notes.SharingExtension -> If this is apple notes, can we have the channel say 'Apple Notes' ?